easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.6-iomkl-2018a-R-3.4.3.eb

```diff
@@ -0,0 +1,455 @@
+easyblock = 'Bundle'
+
+name = 'R-bundle-Bioconductor'
+version = '3.6'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
+
+dependencies = [
+    ('R', '3.4.3', '-X11-20171023'),
+    ('HDF5', '1.10.1'),  # for rhdf5
+    ('MariaDB-connector-c', '2.3.4'),  # for RMySQL
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/3.6/bioc/src/contrib/',
+        'https://bioconductor.org/packages/3.6/data/annotation/src/contrib/',
+        'https://bioconductor.org/packages/3.6/data/experiment/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz'
+}
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+# CRAN packages on which these Bioconductor packages depend are available in R module on which this depends
+# !! order of packages is important !!
+# packages updated on Dec 19th 2017
+exts_list = [
+    ('BiocGenerics', '0.24.0', {
+        'checksums': ['517d31c47d7d807170031e116af3ffeba57baca3980ef4d85b37c2ec3bdc9d0f'],
+    }),
+    ('Biobase', '2.38.0', {
+        'checksums': ['6fa2f0dc214d42f2d0afcc42d2f4da2223ebe67a2f79ff69fba6dc19940cf5b1'],
+    }),
+    ('S4Vectors', '0.16.0', {
+        'checksums': ['8043a0a9f76e00ab96d4214a0ac888d7cf26c0917bb5b3e48ed5503bc6df480f'],
+    }),
+    ('IRanges', '2.12.0', {
+        'checksums': ['a51e7219a9adedcc3ec708db570dbe8ed1156b0bd2bc882fbf5577cad3fa0cef'],
+    }),
+    ('GenomeInfoDbData', '1.0.0', {
+        'checksums': ['d7bb73db4a76960e65c5f6d8ccdcb9480a48cea529a30ea2399f8e4c9887e876'],
+    }),
+    ('GenomeInfoDb', '1.14.0', {
+        'checksums': ['d7db65fe2a494c3a1a0a087a2776eb69cca914024476b920db8f30356b0415ca'],
+    }),
+    ('AnnotationDbi', '1.40.0', {
+        'checksums': ['8c5a98056d4ab3c215d6a01b64953589d127be71940cfd0031f694a382c604b6'],
+    }),
+    ('zlibbioc', '1.24.0', {
+        'checksums': ['22f9bb05a415ae3ce828a904abc0f290aede5ae5fdb212fc75f4c152e08229ff'],
+    }),
+    ('XVector', '0.18.0', {
+        'checksums': ['b36e985344cc3d033d567e2206029e272e1fc23dc5e0ce66c8149ddcdb1c91c4'],
+    }),
+    ('Biostrings', '2.46.0', {
+        'checksums': ['f353f6c1fdbc520a685b880ef05f61245f04397adfcaae4c1395e14e1b06e56d'],
+    }),
+    ('GenomicRanges', '1.30.1', {
```

Comment:
@edmondac It seems like this is already outdated, `GenomicRanges` is at least at `1.30.2` (while `1.30.1` is nowhere to be found now...)